src/commands/api-mesh/status.js

```diff
@@ -44,94 +44,87 @@ class StatusCommand extends Command {
-							);
-							break;
-						default:
-							//Case when deployments table has no status and only initial metadata
-							this.log(
-								'******************************************************************************************************',
-							);
-							this.log('Your mesh status is not available. Please wait for a while and try again.');
-							this.log(
-								'******************************************************************************************************',
-							);
-					}
-					this.log(chalk.blackBright('Legacy Mesh Status:'));
-				}
+				this.log(
+					'******************************************************************************************************',
+				);
 				switch (mesh.meshStatus) {
 					case 'success':
 						this.log(
-							'******************************************************************************************************',
-						);
-						this.log('Your mesh has been successfully built.');
-						this.log(
-							'******************************************************************************************************',
+							`${chalk.bold(`Legacy Mesh:`)} ${chalk.blackBright(`has been successfully built.`)}`,
 						);
 						break;
 					case 'pending':
 						this.log(
-							'******************************************************************************************************',
-						);
-						this.log('Your mesh is awaiting processing.');
-						this.log(
-							'******************************************************************************************************',
+							`${chalk.bold(`Legacy Mesh:`)} ${chalk.blackBright(`is awaiting processing.`)}`,
 						);
 						break;
 					case 'building':
 						this.log(
-							'******************************************************************************************************',
-						);
-						this.log(
-							'Your mesh is currently being provisioned. Please wait a few minutes before checking again.',
-						);
-						this.log(
-							'******************************************************************************************************',
+							`${chalk.bold(`Legacy Mesh:`)} ${chalk.blackBright(
+								`is currently being provisioned. Please wait a few minutes before checking again.`,
+							)}`,
 						);
 						break;
 					case 'error':
-						this.log(
-							'******************************************************************************************************',
-						);
-						this.log('Your mesh errored out with the following error. ', mesh.error);
-						this.log(
-							'******************************************************************************************************',
-						);
+						this.log(`${chalk.bold(`Legacy Mesh:`)} ${chalk.blackBright(`Build has errors.`)}`);
+						this.log(`${chalk.blackBright(mesh.error)}`);
 						break;
 				}
+				if (showEdgeMeshUrl) {
+					if (mesh.meshStatus == 'error') {
+						this.log(`${chalk.bold(`Edge Mesh:`)} ${chalk.blackBright(`Build has errors`)}`);
```

Comment:
This log message is missing a `.` to make it consistent with the other log messages.

Reply:
done.. thanks for your attention to details